src/org/thoughtcrime/securesms/WebxdcActivity.java

```diff
@@ -105,8 +106,12 @@ public boolean onPrepareOptionsMenu(Menu menu) {
   }
 
   @Override
-  protected void openOnlineUrl(String url) {
-    Toast.makeText(this, "Please embed needed resources.", Toast.LENGTH_LONG).show();
+  protected boolean openOnlineUrl(String url) {
+    if (url.startsWith(baseURL +"/")) {
+      // internal page, continue loading in the WebView
+      return false;
+    }
+    return super.openOnlineUrl(url);
```

Comment:
if we want to merge this pr soon for 1.30, i'd be a more reserved here, see comment above.

```suggestion
    Toast.makeText(this, "Please embed needed resources.", Toast.LENGTH_LONG).show();
    return true; // returning `true` causes the WebView to abort loading
```